### ISSUE TITLE
syntax/fetchmail.vim: sslcertfile not highlighted

### ISSUE THREAD

**Issue by jwilk** [vim/vim#10541]
Vim doesn't highlight the [sslcertfile](https://manpages.debian.org/bullseye/fetchmail/fetchmailrc.1.en.html#sslcertfile) keyword in fetchmailrc files, e.g.:
```
poll example.org
    user "jwilk"
    password "hunter2"
    sslcertfile "/usr/share/ca-certificates/mozilla/USERTrust_RSA_Certification_Authority.crt"

# vim:ft=fetchmail
```

**Comment by chrisbra** [comment]
would you create a PR against https://github.com/vim/vim/blob/master/runtime/syntax/fetchmail.vim? I think Nicolai is no longer maintaining vim files, so you may as well add yourself as new maintainer :)

**Comment by jwilk** [comment]
Fixed in 0d878b95d8f9ece2fdba81050f5caba224540f9c.
